src/mscorlib/shared/System/IO/FileLoadException.cs

```diff
@@ -82,12 +84,15 @@ public override string ToString()
         protected FileLoadException(SerializationInfo info, StreamingContext context)
             : base(info, context)
         {
-            throw new PlatformNotSupportedException();
+            FileName = info.GetString("FileLoad_FileName");
+            FusionLog = info.GetString("FileLoad_FusionLog");
```

Comment:
Desktop did try/catch. Does it matter?
```c#
            try
            {
                _fusionLog = info.GetString("FileLoad_FusionLog");
            }
            catch 
            {
                _fusionLog = null;
            }
```

Reply:
Oh it was probably just for the CAS on FusionLog so it's not needed

Reply:
jep, I skipped it.